Frends.CSV.Create/Frends.CSV.Create.UnitTests/UnitTests.cs

```diff
@@ -6,7 +6,8 @@ namespace Frends.CSV.Create.UnitTests;
 [TestClass]
 public class UnitTests
 {
-    private readonly string _xml = @"<?xml version=""1.0""?>
+    private readonly string _xml =

```

Comment:
I know its hard, but when doing separate tickets try to not alter the other existing code :) At least if it is not really necessary. This make reviews harder and then a changeset about one feature would contain things that touch other features.